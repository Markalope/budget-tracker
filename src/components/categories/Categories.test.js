import React from 'react';
import { shallow } from 'enzyme';
import Categories from './Categories';
import toJSON from 'enzyme-to-json';

describe('Categories', () => {
  it.skip('renders categories', () => {
    const categories = [
      { key: '1', name: 'a' },
      { key: '2', name: 'b' },
      { key: '3', name: 'c' }
    ];

    const wrapper = shallow(
      <Categories categories={categories} />
    );

    expect(toJSON(wrapper)).toMatchSnapshot();
  });

});