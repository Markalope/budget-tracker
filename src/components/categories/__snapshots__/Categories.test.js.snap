// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Categories renders categories 1`] = `
<ul>
  <Connect(Category)
    category={
      Object {
        "id": "1",
        "name": "a",
      }
    }
    id="1"
  />
  <Connect(Category)
    category={
      Object {
        "id": "2",
        "name": "b",
      }
    }
    id="2"
  />
  <Connect(Category)
    category={
      Object {
        "id": "3",
        "name": "c",
      }
    }
    id="3"
  />
</ul>
`;
